FBSnapshotTestCase/FBSnapshotTestController.h

```diff
@@ -85,6 +85,16 @@ extern NSString *const FBDiffedImageKey;
  */
 @property (readwrite, nonatomic, copy) NSString *referenceImagesDirectory;
 
+/**
+ The name of the tests. This will be used to create folders with the specified name.
```

Comment:
I guess there's probably a naming issue around here. 

Internally `testName` refers to the class name rather than the test case itself (if we don't set it, it produces the folder name equal to the name of the test file). So, one thing is for sure, I should definitely change the property name in the `FBSnapshotTestCase` class which is the one available in the API. 

I kept `testName`because it's what was being used internally before, but I guess now, being open to the client it's more meaningful to name it something else. 

What would you think about `folderName`, or even `testClass`... I'm open to suggestions. :) 
I like `folderName` because it's very explicit on what it's going to do. I'm going with that now, and if you'd like something else we can discuss it later. :)